v0.15.0.0-linux/dikdust/monero-linux-0.15-build.assert

```diff
@@ -0,0 +1,568 @@
+--- !!omap
+- out_manifest: |
+    369442e10cdb31b0fe528d66e4d6cf1df297cabb01195204553aa3a6564ae992  monero-aarch64-linux-gnu-v0.15.0.0.tar.bz2
```

Comment:
Seems we have a different hash here:
* `369442e10cdb31b0fe528d66e4d6cf1df297cabb01195204553aa3a6564ae992`
VS:
* `f92f0acbc49076ad57337b5928981cd72c01aabe6a8eb69a1782f7fa1388fb77 `
https://github.com/monero-project/gitian.sigs/blob/master/v0.15.0.0-linux/TheCharlatan/monero-linux-0.15-build.assert#L3

Reply:
REPOSITORY                   TAG                 IMAGE ID            CREATED             SIZE
base-bionic-amd64            latest              7583fdaf398d        44 hours ago        974MB

Reply:
REPOSITORY                   TAG                 IMAGE ID            CREATED             SIZE
base-bionic-amd64            latest              7583fdaf398d        44 hours ago        974MB

Reply:
Yep, seems the image has changed since we all built previous versions.
`base-bionic-amd64` was at `63c244a6a71b` then: https://github.com/monero-project/gitian.sigs/pull/23

We really need to find a way to lock those down. :-(

Would you be able to switch to that docker image?

@TheCharlatan Any other ideas / pointers?

Reply:
I'd suggest removing the `v0.15.0.0` sigs from this PR while sorting out the diffs.